tasks/main.yml

```diff
@@ -11,6 +11,7 @@
     version: "{{ gie_proxy_git_version |default(omit) }}"
   notify:
     - Restart GIE Proxy
+  when: gie_proxy_install
```

Comment:
Can we rename GIE everywhere to the new name?

Reply:
Sure, but we might not be the only users using this repo to install the Gx-IT proxy and therefore changing the names would break things everywhere.